themes/ssp/core/no_metadata.tpl.php

```diff
@@ -1,9 +1,8 @@
 <?php
 
-echo('<p>' . $this->t('{core:no_metadata:config_problem}') . '</p>');
-
-echo('<ul class="list-unstyled">');
-echo('<li>' . $this->t('{core:no_metadata:suggestion_user_link}') . '</li>');
-echo('<li>' . $this->t('{core:no_metadata:suggestion_developer}') . '</li>');
-echo('</ul>');
+echo ('<p>' . $this->t('{core:no_metadata:config_problem}') . '</p>');
 
```

Comment:
The same comment as below. Remove all echo.

Reply:
Resolved by commit [Remove sequentially echos ](https://github.com/EGI-Federation/simplesamlphp-module-themeegi/pull/14/commits/0d8e00ed36955b6ffb716baef99a40a281115645)